devel/gcli/Portfile

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem            1.0
+
+PortGroup             github 1.0
+
+name                  gcli
```

Comment:
You don't need to set `name` and `version` because `github.setup` will define that implicitly.  You can delete your `name` and `version` lines, and move `githup.setup` up here:

`github.setup          herrhotzenplotz gcli 2.3.0 v`